profanity_word_phrase_filter/README.md

```diff
@@ -1,19 +1,20 @@
 ## Custom profanity filter
 
-Most online platforms contain inappropriate language. Parent's want to be certain
+Most online platforms contain inappropriate language. Parents want to be certain
 that Story squad (aka scribble stadium) is offering a safe environment for their
```

Comment:
Squad should be capitalized as it is part of the company name. Scribble Stadium should also be capitalized, as it is the name of the App developed by Story Squad.

Reply:
Correction made.